great_expectations/core/validation_config.py

```diff
@@ -23,9 +22,26 @@ class ValidationConfig(BaseModel):
 
     """
 
+    class Config:
+        arbitrary_types_allowed = True
+        json_encoders = {
+            ExpectationSuite: lambda v: expectationSuiteSchema.dump(v),
+        }
+
     name: str
     data: BatchConfig  # TODO: Should support a union of Asset | BatchConfig
     suite: ExpectationSuite
+    id: Union[str, None] = None
+
+    @validator("suite", pre=True)
+    def _validate_suite(cls, v):
```

Comment:
same as above - I think it's worth a docstring explaining this is just because the suite isn't pydantic.